src/rgw/driver/d4n/d4n_directory.h

```diff
@@ -24,78 +26,97 @@ struct CacheObj {
 
 struct CacheBlock {
   CacheObj cacheObj;
-  std::string blockId; /* block ID */
+  std::string version; /* RADOS object block ID */
```

Comment:
Why has the blockId been removed? Please re-instate it and set it to the value used in the PR. Also let the version be separate.

Reply:
The blockID is meant to be the unique identifier for the block and originally, in the D4N code, it was described as the "RADOS object block ID". As a result, my understanding is that this is the version, especially since it was used in `build_index` to create the key for the directory entries.